README.md

```diff
@@ -1 +1,69 @@
-# datatable
\ No newline at end of file
+![Linting](https://github.com/crossroadscx/datatable/actions/workflows/linting.yml/badge.svg) ![Chromatic](https://github.com/crossroadscx/datatable/actions/workflows/chromatic.yml/badge.svg)
+# React Table DataTable
+
+This project layers a UI on top of the every popular [React Table](https://react-table.tanstack.com/) library. The intention is to have a drop-in UI solution for most table use cases, while including simple options to override things like theming, sub-components, and actions.
+## Demo
+
```

Comment:
Would be nice if we add some screenshots of the component and may be some styling applied!.

Reply:
Absolutely agree. We don't have the theming capability built in yet, but will include examples when that comes in. I will add some screenshots

Reply:
Resolved in 990c310d84e844d64909725416c51acd2cd89787